documentation/projects/proposals/additional_search_views/20230424-project_proposal_additional_search_views.md

```diff
@@ -0,0 +1,36 @@
+# 2023-04-24 Project Proposal
+
+## Reviewers
+
+- [ ] @obulat
+- [ ] @krysal
+
+## Project summary
+
+The Openverse team has decided to enhance the browsing experience by
+incorporating new content views for providers, tags, and creators. This feature
+will allow users to share collections detached of individual searches.
+
+## Goals
+
+Search Experience
+
+## Requirements
+
```

Comment:
I remember we discussed something on our last meetup related to the navigation with tags, like how the terms in tags will apply in the search, as `or` or `and` logical operations? Do you think is worth adding those here as requirements for the project? It would be helpful to know how they would be applied and to limit ourselves to specific functionalities :)

Other points that could be specified here:
- will the tags from the current details page be clickable to navigate to these new views?
- the new views will include the filter sidebar?
- the new views will show the lastet images/audio added to Openverse first?

Reply:
Just to give info for the current implementation, right now field specific queries (tags, creator, title) use ["simple query string"](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/query-dsl-simple-query-string-query.html) style query which defaults to "OR".

Reply:
I suggest to add the following requirements, based on my understanding of what this project is.

The project creates 3 additional views for the Nuxt frontend.
1. View by tags
The page shows the name of the tag, and all the items of the selected media type (image/audio) that have this exact tag applied (so, for `cat` tag view, only images that have a tag `cat`, not `cats` will be shown). The items are sorted by `ingested_on` field in descending order. The user can go to this view directly, by opening www.openverse.org/tags/image/cat or by clicking on one of the tags on the single result page.

2. View by provider
The page shows the name of the provider, [with their logo and description? Or would that be out of scope for this project?], and all the items from that provider sorted by `ingested_on` field in descending order. 
The user can go to this view directly, by opening www.openverse.org/providers/wikimedia/image/ or by clicking on the provider's name on the single result page.
Note: for providers, I'm not sure what we want to do, so here are some questions: 
- Do we want to use `providers` or  `sources`? The difference is that provider is the *site* where we got the image from, and the source _can_ be the same as the provider, but can also be the collection on that provider. For example, NASA publishes images on Flickr. For these images, `source` is NASA, and the provider is Flickr. Naturally, there are more sources than providers. Do we want to create views for all sources, or only for providers?
- For providers with multiple media types, such as Wikimedia or Europeana, do we want to have an "All content" veiw, or only the separate image/audio views?

3. View by creator
The page shows the name of the creator (together with the provider? otherwise the creators will clash because different creators can use the same name on different providers), and all the items from that creator sorted by `ingested_on` field in descending order. The user can go on this view directly, by opening www.openverse.org/creators/cat_flickr (for a creator named "cat" on Flickr), or by clicking on the creator's name on the single result page.

These views will require changes to:
- Nuxt frontend: adding pages for the specific views. They can reuse the grids and result boxes/squares from the search views.  The pages will only show the results, without the ability to search within them. Because of that, the views will use the "content" header (without the search bar, search type selection or filters button) and will not have a filter sidebar.

- API: adding `filter` parameters that would enable retrieving all items of a certain media type that have the specific property (tag/provider/creator and source pair). These will work similarly to the way filter parameters such as `license` work now, returning only the items with the specific filter. Searching within the filtered items is not in scope of this project.